cnames_active.js

```diff
@@ -2870,6 +2870,7 @@ var cnames_active = {
   "soda": "yieldray.github.io/soda",
   "solace": "cname.vercel-dns.com", // noCF
   "solder": "mcrocks999.github.io/solder.js",
+  "solitude": "cname.vercel-dns.com.",
```

Comment:
```suggestion
  "solitude": "cname.vercel-dns.com", // noCF
```
(not sure why the CI didn't detect this)
If you're using Vercel, you'll want the Cloudflare proxy disabled